ehrapy/preprocessing/_imputation.py

```diff
@@ -2,22 +2,18 @@
 
 import warnings
 from collections.abc import Iterable
-from typing import TYPE_CHECKING, Literal
+from typing import TYPE_CHECKING, List, Literal
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Literal
```

We are using 3.10+ type hints. We don't import primitive types anymore.

`from __future__ import annotations` allow us to use that feature in a forward compatible way.